pydicom/config.py

```diff
@@ -292,6 +292,27 @@ def needs_to_convert_to_RGB(ds):
 values within the dataset when applying corrections.
 """
 
+WARN_ON_INVALID_KEYWORD = True
+"""Warn when setting a :class:`~pydicom.dataset.Dataset` camel case attribute
```

Comment:
Point of discussion:  I had argued before to make this multi-valued (e.g. "IGNORE", "WARN", "ERROR"), not just boolean, so it could be turned into an _error_ in _pydicom_ 3.x, not just a warning.  Thoughts?

Reply:
No, that makes sense. Sorry, I completely overlooked that